assets/shaders/custom_vertex_attribute.wgsl

```diff
@@ -1,28 +1,29 @@
 #import bevy_pbr::mesh_view_bindings
 #import bevy_pbr::mesh_bindings
 
-struct Vertex {
-    [[location(0)]] position: vec3<f32>;
-    [[location(1)]] blend_color: vec4<f32>;
-};
-
 struct CustomMaterial {
     color: vec4<f32>;
 };
 [[group(1), binding(0)]]
 var<uniform> material: CustomMaterial;
 
+// NOTE: Bindings must come before functions that use them!
+#import bevy_pbr::mesh_functions
```

Comment:
What is the reasoning for this import statement to not be at the top of the file like the rest?

Reply:
In my mind, the comment is directly about the position of this one import in relation to the others and the types and bindings in the file. As such, it made sense to me to add the note on it.

Reply:
Yes the comment I get. To clarify, why this:

```
#import X
[bindgroup(A)]

// Some comment
#import Y
[bindgroup(B)]
```
Instead of this:
```
#import X;
// Some comment
#import Y;

[bindgroup(A)]

[bindgroup(B)]
```

Reply:
That’s what the comment is supposed to explain. Types and bindings must come before functions that use them. The import statement that the comment is about is importing ‘bevy_pbr::mesh_functions’ which contains a bunch of helper functions and those functions refer to members of the view and mesh bindings.

Reply:
I think your comment is both clear and understandable :) Placing the comment + import statement and the usage of the function makes completely sense in this case. 

This is more me trying to understand wgsl (new to me). I was curious that perhaps there was something in wgsl that _required_ the import statement to placed close to where it was going to be used, but I understand now that as long as it is placed before, it could be placed anywhere.